Books/client/src/components/Wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,75 @@
+import './Wrapper.scss';
```

Comment:
I think this component should be renamed to something more readable because I can't really figure out what this component is for without poking around

Reply:
Just changed it to OuterPage, is it better? I'm truly not sure which name to give that component, do you have suggestions?

Reply:
You could name it Dashboard since that is essentially what the user would see whenever they log in

Reply:
Sure, just did it